go-fuzz/versifier/versifier.go

```diff
@@ -571,8 +571,8 @@ func tokenize(data []byte) []Node {
 func structure(nn []Node) []Node {
 	nn = extractNumbers(nn)
 	nn = structureBrackets(nn)
-	nn = structureLists(nn)
 	nn = structureKeyValue(nn)
+	nn = structureLists(nn)
```

Comment:
Why is this better?
There can be lists of kv and kv with lists. For example, http headers can be kv with lists in value.